### ISSUE TITLE
overlay does not fill the whole screen

### ISSUE THREAD

**Issue by msevestre** [santomegonzalo/react-native-floating-action#35]
You can see the behavior in your example. The header bar is not covered by the overlay. Is this on purpose?

**Comment by santomegonzalo** [comment]
@msevestre this is a correct behavior because the overlay will be on your "scene". Using react-navigation the "scene" be between **NavigationBar** and **TabBar**. I think it's not possible to do show the overlay on top of everything.

**Comment by msevestre** [comment]
Looking at inbox from Google, the overlay covers everything but the toolbar. That would be what I would expect here as well.

What about using width and height of screen? I could give it a try if you agree that this could be a good feature.

**Comment by santomegonzalo** [comment]
of course, you are more than welcome to try. But still, I think will not work because of how react-native works.  
Let me know.

**Comment by santomegonzalo** [comment]
@msevestre I tried what you mention and will not work. Its a limitation of react-native. On another project I was trying to open a popup from a button, mounting it with position: absolute and everything but the popup was just inside the button. 

There is a way to do it and I'm testing it to see how works but I will need to change a lot and could break current implementations. I need to keep investigating and testing it... this is so far what i have

![mar-29-2018 10-55-15](https://user-images.githubusercontent.com/2914973/38079639-b5649616-333f-11e8-8ecf-73f5fec253a0.gif)

but now my buttons are behind the background... 

if you have another idea please let me know... otherwise I will keep you posted

**Comment by msevestre** [comment]
@santomegonzalo Yeah thinking about it a bit more, I understand the problems you were mentioning.

I wonder if something like [that](https://github.com/testshallpass/react-native-dropdownalert/blob/master/DropdownAlert.js#L471) could be used. Somehow the alert comes on top of everything and can be declared anywhere in the tree.

**Comment by fukemy** [comment]
hi, does this problem fixed? I have navigationbar + bottom navigation, so the action view does not cover all => UI seem not good
